types/slate/index.d.ts

```diff
@@ -325,6 +325,7 @@ export class Text extends Immutable.Record({}) {
     key: string;
 
     readonly text: string;
+    readonly marks: Immutable.List<Mark>;
```

Comment:
Should this be `Immutable.Set<Mark>` instead? I know the documentation says it's a list, but it looks like it's probably a set judging by the code: https://github.com/ianstormtaylor/slate/blob/67e397100e2f4cc6bcd4b8f98d7a85dcf30c72fd/packages/slate/src/models/text.js#L98

Reply:
You're right.  I was going based on TextProperties, but looking at Leaf class, marks are `Immutable.Set<Marks>`.  Changed.